front/src/shared.module.scss

```
@@ -0,0 +1,168 @@
+@import './colors.module.scss';
+
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: #d6d1bd;
+  /* enable in development
+     background-color: pink; */
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
+
+.layout-row {
+  display: flex;
+  flex-direction: row;
+}
+
+.layout-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.arrow {
+  @extend .layout-row;
+  align-items: center;
+  width: 0;
+  height: 0;
+  border-top: 20px solid transparent;
+  border-bottom: 20px solid transparent;
+  cursor: pointer;
+}
+
+.arrow-right {
+  @extend .arrow;
+  margin-left: 5vw;
+  float: right;
+  border-left: 20px solid $COLOR_BLACK;
+}
+.arrow-left {
+  @extend .arrow;
+  margin-right: 5vw;
+  text-align: right;
+  border-right: 20px solid $COLOR_BLACK;
+}
+
+// FROM INFOBOXES
+
+.infoItem {
+    display: flex;
+    min-width: 320px;
+    width: 30vw;
+    height: 38px;
+    justify-content: left;
+}
+
+.box {
+    float: left;
```

css.box is always used with css.noFlex in dayview and infoboxes, thus .box should be extended with .noFlex (and css.noFlex's should be removed from classnames with css.box)
```suggestion
.box {
    @extend .noFlex;
    float: left;
```